lib/shoryuken/environment_loader.rb

```diff
@@ -15,9 +15,9 @@ def initialize(options)
     end
 
     def load
-      initialize_logger
       load_rails if options[:rails]
       Shoryuken.options.merge!(config_file_options)
+      initialize_logger
```

Comment:
@h3poteto I'm wondering what we can do with [this log](https://github.com/crowdworks/shoryuken/blob/fix-initialize-logger/lib/shoryuken/environment_loader.rb#L107) - with the change above it goes to the `stdout`, right?